src/app/template-library/template-library.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="ion-page" style="background: #ffffff">
+<ion-split-pane class="template-library-container" contentId="main">
+  <ion-menu contentId="main" style="border: none">
+    <div class="search-home-container">
+      <ion-searchbar color="light" class="template-search-bar"></ion-searchbar>
```

Comment:
There should be a 'home' button before the search bar

Reply:
Talking with Phil he decided not to include it. He hasn't updated the design yet